draft-ietf-rats-architecture.md

```diff
@@ -133,6 +123,10 @@ Attester:
 
 : An entity whose attributes must be appraised in order to determine whether the entity is considered trustworthy, such as when deciding whether the entity is authorized to perform some operation
 
+Claim:
+: A piece of asserted information, often in the form of a name/value pair.
+(Compare /claim/ in {{RFC7519}})
```

Comment:
"Note: use of 'claim' in this document differs from RFC7519 in that this document doesn't imply a particular encoding"